modules/authentication/portal-authentication-oauth/src/main/java/de/cuioss/portal/authentication/oauth/impl/Oauth2ServiceImpl.java

```diff
@@ -119,19 +119,35 @@ public void filter(ClientRequestContext requestContext) {
 
     @Override
     public AuthenticatedUserInfo createAuthenticatedUserInfo(final HttpServletRequest servletRequest,
-                                                             final UrlParameter code, final UrlParameter state, final String scopes, final String codeVerifier) {
+            final UrlParameter code, final UrlParameter state, final String scopes, final String codeVerifier) {
+
         requireNonNull(servletRequest);
         requireNonNull(code);
         requireNonNull(state);
         requireNonNull(emptyToNull(scopes));
+
         var configuration = configurationProvider.get();
         Token token;
+
         final var builder = new CuiRestClientBuilder(log)
             .basicAuth(configuration.getClientId(), configuration.getClientSecret())
             .register(new AcceptJsonHeaderFilter());
-        try (final var requestToken = builder.url(configuration.getTokenUri().trim()).build(RequestToken.class)) {
-            token = requestToken.requestToken("authorization_code", code.getValue(), state.getValue(), codeVerifier,
-                configuration.getExternalContextPath().trim() + servletRequest.getRequestURI());
+
+        final String tokenUri = configuration.getTokenUri().trim();
+        final String redirectUri = configuration.getExternalContextPath().trim() + servletRequest.getRequestURI();
+        log.debug("creating auth user info with scopes='{}', tokenUri='{}', redirectUri='{}'",
```

Comment:
maybe this should be TRACE due to sensible data in the tokenuri like the `nonce`?!

Reply:
In general an excellent finding, in the concrete example there is no sensitive information: Only Scopes and uris